udata/tests/api/test_datasets_api.py

```diff
@@ -69,13 +93,13 @@ def test_dataset_api_list_filtered_by_org(self):
             VisibleDatasetFactory()
             dataset_org = VisibleDatasetFactory(organization=org)
 
-        response = self.get(url_for('api.datasets'),
+        response = self.get(url_for('apiv2.dataset_search'),
```

Comment:
Should we use Mongo here instead of ES? It is not really a search but a matching query isn't it? @maudetes